trace/spanstore.go

```diff
@@ -195,6 +137,64 @@ func SampledSpansSummary() map[string]PerMethodSummary {
 	return out
 }
 
+// ReportSpansByLatency returns a sample of successful spans.
+//
+// minLatency is the minimum latency of spans to be returned.
+// maxLatency, if nonzero, is the maximum latency of spans to be returned.
+func ReportSpansByLatency(name string, minLatency, maxLatency time.Duration) []*SpanData {
```

Comment:
I would be OK removing this functionality until we find the right home for it.

Reply:
+1.